src/main/scala/slamdata/Publish.scala

```diff
@@ -0,0 +1 @@
+../../../../project/Publish.scala
```

Comment:
What does this mean? Is it just GitHub’s way of showing me a symlink or something?